### PR TITLE
Fix patchmatch-docs

### PR DESCRIPTION
**Pull request by mauwii**
Fixing the formatting of patchmatch docs and 2 broken links

### CODE REVIEW

**Comment by mauwii** [comment]
Just realized after opening this PR that it includes the change from #2110 as well 🤦🏻‍♂️

**Comment by mebelz** [comment]
You might want to link to 
https://github.com/invoke-ai/InvokeAI/discussions/1893
for Patchmatch on Mac

**Comment by mauwii** [comment]
No, since this would not be necesarry if my [PR in invoke-ai/PyPatchMatch](https://github.com/invoke-ai/PyPatchMatch/pull/6) gets merged 😜

File: docs/installation/060_INSTALL_PATCHMATCH.md [hunk 1]
```diff
@@ -2,114 +2,111 @@
...
-   [dep] csrc/pyinterface.cpp ...
-   [CC] csrc/pyinterface.cpp ...
-   [CC] csrc/inpaint.cpp ...
-   [CC] csrc/nnf.cpp ...
-   [CC] csrc/masked_image.cpp ...
-   [link] libpatchmatch.so ...
-   ```
-
+    ```sh
+    pip install "git+https://github.com/invoke-ai/PyPatchMatch@0.1.3#egg=pypatchmatch"
```
Comment (lstein) [inline docs/installation/060_INSTALL_PATCHMATCH.md:55]: If you really intend to do a "develop" install, as stated in line 52, then the user will need to do `pip install -e "git+https://....` However, this is no longer necessary.

File: docs/installation/060_INSTALL_PATCHMATCH.md [hunk 2]
```diff
@@ -2,114 +2,111 @@
...
+    cd /usr/lib/x86_64-linux-gnu/pkgconfig/
+    ln -sf opencv4.pc opencv.pc
+    ```
 
-4. Activate the environment you use for invokeai, either with
-`conda` or with a virtual environment.
+4. Activate the environment you use for invokeai, either with `conda` or with a
+   virtual environment.
 
 5. Do a "develop" install of pypatchmatch:
```
Comment (lstein) [inline docs/installation/060_INSTALL_PATCHMATCH.md:52]: I don't think this is necessary with the current version of pypatchmatch. The regular install works.

**Review by lstein — CHANGES_REQUESTED** [review]
Please confirm that "develop" mode is still needed for pypatchmatch installation. I don't think it is.

File: docs/installation/060_INSTALL_PATCHMATCH.md (hunk 1 above)
Comment (mauwii) [reply docs/installation/060_INSTALL_PATCHMATCH.md:55]: I did not change any content, just fixed the broken format

File: docs/installation/060_INSTALL_PATCHMATCH.md (hunk 2 above)
Comment (mauwii) [reply docs/installation/060_INSTALL_PATCHMATCH.md:52]: Same here, I did not change content, just fix the format.

File: docs/installation/060_INSTALL_PATCHMATCH.md (hunk 1 above)
Comment (lstein) [reply docs/installation/060_INSTALL_PATCHMATCH.md:55]: got it. Fixing it on my end.

**Review by lstein — APPROVED** [review]
Thanks!

**Pull request merged**